### PR TITLE
Neutralise BT on X9E, unless DEBUG is defined

### PR DESCRIPTION
**Pull request by 3djc**
This is WIP but brings things is a somewhat 'releasable' state :)

### CODE REVIEW

File: radio/src/bluetooth.cpp [hunk 1]
```diff
@@ -64,6 +64,7 @@ char * bluetoothReadline()
 
   while (1) {
     if (!btRxFifo.pop(byte)) {
+      TRACE("NO RESPONSE FROM BT MODULE");
```
Comment (bsongis) [inline radio/src/bluetooth.cpp]: It will fire a lot of traces, no?

File: radio/src/opentx.h [hunk 2]
```diff
@@ -194,7 +194,7 @@
   #define CASE_PCBX9E(x)
 #endif
 
-#if defined(BLUETOOTH)
+#if defined(BLUETOOTH) && !(defined(PCBX9E) && !defined(DEBUG))
```
Comment (bsongis) [inline radio/src/opentx.h]: difficult to read!

```
#if defined(BLUETOOTH) && (!defined(PCBX9E) || defined(DEBUG))
```

File: radio/src/bluetooth.cpp (hunk 1 above)
Comment (3djc) [reply radio/src/bluetooth.cpp]: It did not at all on X9E, only when sending a CLI command to module and module would not respond. Will do some tests with Horus

Comment (3djc) [reply radio/src/bluetooth.cpp]: Given I cannot test with two radio connected, I made a change that should ease your concern

**Comment by bsongis** [comment]
👍

**Pull request merged**